api/instance/v1/instance_sdk.go

```diff
@@ -4125,11 +4139,18 @@ type SetIPRequest struct {
 	Organization string `json:"organization"`
 
 	Tags []string `json:"tags"`
+
+	Project string `json:"project"`
 }
 
 func (s *API) setIP(req *SetIPRequest, opts ...scw.RequestOption) (*setIPResponse, error) {
 	var err error
 
+	if req.Project == "" {
```

Comment:
this is an issue, it should be a oneof

Reply:
`setIP` is not used anymore, I will fix it in another MR